### ISSUE TITLE
Older versions of macos

### ISSUE THREAD

**Issue by fiveNinePlusR** [aardappel/treesheets#323]
Is there a technical reason that you don't build for anything but the latest mac os versions for release? Seems like it should work on the older ones if the build targeted earlier versions.

**Comment by aardappel** [comment]
No, mostly.. simplicity? I don't have time to spend more than the absolute minimum supporting platforms, versions, package managers, etc.

If there is a way to have the current Xcode project have settings for multiple OSX builds, AND have those builds be triggered from CI, AND added to releases, AND not slowing down CI significantly (sharing wxWidgets sub-build) AND someone wants to add and test it.. then we can make it happen.

No idea how well wxWidgets still supports older OSX.

**Comment by aardappel** [comment]
Would you like to try and test this, to make sure the above requirements are met?

**Comment by nekoniaow** [comment]
Absolutely.

Note that this is actually a bit more involved if one wants to minimize the (unnecessary) changes Xcode makes to the project when following the above procedure.
I now locally what have what I think is a minimally modified setup which builds two targets:

- TreeSheets.app
- TreeSheets 10.14.app

when building the project.

Both targets use the same build settings except for the target OS version so they will be properly depending on the same installed wxWidget version.
I have yet to verify the build locally because I haven't installed wxWidgets locally yet but I will do so tomorrow and verify that I can obtain valid build outputs.

My system runs 10.14 so I will be able to verify that the resulting app runs.

**Comment by aardappel** [comment]
nice, good start!

**Comment by nekoniaow** [comment]
> We can generate macOS builds now with CMake and the underlying build system.

Note though that this issue was about old versions of MacOS, are those versions supported with CMake?

**Comment by tobiolo** [comment]
The builds provided in the Release section on the GitHub Page are compatible down to 10.15.

**Comment by tobiolo** [comment]
You are welcome!
